templates/skeleton/app/routes/account/login.tsx

```diff
@@ -74,3 +66,25 @@ export default function Login() {
     </Form>
   );
 }
+
+// Remix v1 implementation of ErrorBoundary; for Remix v2, this will need to be updated according to the following documentation:
+// https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
+export const ErrorBoundary: ErrorBoundaryComponent = ({error}) => {
+  console.error(error);
+
+  return <div>There was an error.</div>;
+};
+
+// Remix v1 implementation of CatchBoundary; for v2, you can remove `CatchBoundary` and implement the error handling in `ErrorBoundary`:
+// https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
```

Comment:
Same here:

```ts
/**
 * Remix v1 implementation of CatchBoundary. This will be removed in a future release
 * in favor of the Remix v2 ErrorBoundary component.
 * @see https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
 */
```